### PR TITLE
Created test cases for PositionDBConnector

### LINKED ISSUE #109: Create test cases for PositionDBConnector
1. Read [guidelines](https://gist.github.com/djeck1432/f722f5fe518c80b248ff609627da25e5).
2. Add positive and negative test cases for class methods [PositionDBConnector](https://github.com/djeck1432/spotnet/blob/main/web_app/db/crud.py#L160)
3. Use please mock connection to db from [confest.py](https://github.com/djeck1432/spotnet/blob/main/web_app/tests/conftest.py)
4. Make sure you have run successfully `pylint` command without any issues 
`pylint $(git ls-files '*.py') --disable=all --enable=C0114,C0115,C0116,C0301`

### PR DESCRIPTION
**Pull request by Jemiiah**
Fixes #109

### CODE REVIEW

File: web_app/tests/test_PositionDBConnector.py [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+        start_price=0.0
+    )
+
+
+@pytest.fixture(scope="function")
+def mock_session():
+    """Fixture to create a mock database session."""
+    session = MagicMock()
+    return session
+
```
Comment (djeck1432) [inline web_app/tests/test_PositionDBConnector.py:37]: remove empty blanks, between functions should be 2 blanks

File: web_app/tests/test_PositionDBConnector.py [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+        multiplier=3,
+        status=Status.PENDING.value,
+        start_price=0.0
+    )
+    mock_position_db_connector.write_to_db.return_value = new_position
+    mock_position_db_connector.create_position.return_value = new_position
+    
+    result = mock_position_db_connector.create_position(
+        wallet_id="test_wallet_id",
+        token_symbol="BTC",
```
Comment (djeck1432) [inline web_app/tests/test_PositionDBConnector.py:98]: I don't think that we use `BTC` anywhere in the code, please use some values from TokenParams.ETH.name

**Review by djeck1432 — CHANGES_REQUESTED** [review]

**Comment by djeck1432** [comment]
@Jemiiah Any update? 
please update your code until next review tomorrow morning

**Comment by Jemiiah** [comment]
Done

**Pull request merged**